src/app/components/Text/index.jsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
+import remark from 'remark';
+import reactRenderer from 'remark-react';
 import { arrayOf, shape, string } from 'prop-types';
 
 const Text = ({ blocks }) =>
   blocks.map(({ blockId, model }) => (
-    <p key={blockId}>
-      {model.text}
-    </p>
+    <div key={blockId}>
```

Comment:
This usage generates many unnecessary divs surrounding paragraphs. It would be good to see if you can use a `Fragment` instead of a `div` and to see whether you can pass the key onto the generated component below.

Reply:
I agree, there are two many divs. This seems to be because 1) We are already wrapping the parsed content in a div (We couldn't get around this Friday afternoon, as we were not creating custom components and we needed to have a custom react key around the element.)

There may be more things to explore with React-Remark as this library seems to wrap each Element it parses in a DIV and then puts the O tag or whatever it might be inside of that